src/command_line/search/cli_structure_search.rs

```diff
@@ -12,11 +12,18 @@ pub fn cli_structure_search(method: &str, matches: &ArgMatches) -> eyre::Result<
     let smiles = matches
         .get_one::<String>("smiles")
         .ok_or(eyre::eyre!("Failed to extract SMILES"))?;
+    let use_chirality = matches.get_one::<String>("use-chirality");
     let result_limit = matches.get_one::<String>("result-limit");
     let tautomer_limit = matches.get_one::<String>("tautomer-limit");
     let extra_query = matches.get_one::<String>("extra-query");
     let use_scaffolds = matches.get_one::<String>("use-scaffolds");
 
+    let use_chirality = if let Some(use_chirality) = use_chirality {
+        !matches!(use_chirality.as_str(), "false")
```

Comment:
Remind me, what does `matches!` do and where does it come from?

Reply:
`matches!(thing, "true")` is the equivalent of:

`match thing {`
`    "true" => true,`
`    _ => false,`
`}`